src/main/kotlin/minesweeper/domain/strategy/MineDeployStrategy.kt

```diff
@@ -0,0 +1,7 @@
+package minesweeper.domain.strategy
+
+import minesweeper.domain.cell.Coordinate
+
+fun interface MineDeployStrategy {
```

Comment:
```suggestion
interface MineDeployStrategy {
```
코틀린에서는 interface 안에 하나의 함수만 있으면 SAM을 지원하기에 제거하셔도 좋을 것 같아요.

Reply:
오... [문서](https://kotlinlang.org/docs/fun-interfaces.html)에서는 해당 내용을 못본거 같은데.. 👍 
그런데 저는 SAM이라는 걸 보다 더 명시적으로 표현해주고 싶어서 fun 키워드를 사용했습니다!